client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -134,14 +134,18 @@ export const AppSchemaForm = ({ app }) => {
     const matchingHost = Object.keys(state.allocations.hosts).find(
       host => app.resource === host || app.resource.endsWith(`.${host}`)
     );
+    const host = state.allocations.hosts[state.systems.storage.defaultHost];
+    const hasCorral =
+      host === 'cloud.corral.tacc.utexas.edu' || 'data.tacc.utexas.edu';
```

Comment:
We need this same check in the [backend](https://github.com/TACC/Core-Portal/blob/main/server/portal/apps/datafiles/views.py#L101)